config/schemapatch/0000_03_authorization-openshift_01_rolebindingrestriction.crd.yaml

```diff
@@ -0,0 +1,205 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@damemi why is this happening?

Reply:
that this file is being added here? That is weird, @stlaz was this entire file generated, and how?

Reply:
@damemi this file was autogenerated, I have the legacy protobuf 3.0.0 in the api/protoc folder and just run `PATH=$(pwd)/protoc/bin:$PATH make update`

Reply:
hm, nvm, when I removed it, it would not be autogenerated again, perhaps I had dirty directory before